_ClientSideUI/index.html

```diff
@@ -336,14 +336,14 @@
     var metricsDef;        //holds the definition of the metrics (from server side metrics.js)
     var eventsDef;
     var showHiddenNodes=false, showRawSend=false;
-    var socket = io.connect('<?php echo $_SERVER['HTTP_HOST']; ?>', {'connect timeout': 1000}); //limit chrome xhr-polling fall back delay
```

Comment:
That would be fine if it works, I would need to double check this on my own gateway. I plan to make some changes, perhaps I will include the testing of this.